content/ember-cli/v5/travis-ci-support.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Travis CI Support
+since: 5.4.0
+until: 6.0.0
+---
+
+Travis CI is no longer recommended.
```

Comment:
Is there any way that we can make this description a bit longer? 🙈 I don't know if there is an RFC or something that we can pull a blurb from but the fact that this is so short is causing the 5.x page to look a bit strange: https://deploy-preview-1338--ember-deprecations.netlify.app/ember-cli/v5.x/#toc_travis-ci-support 🙈

Reply:
I think adding a min-width is a good thing regardless! The description is just a placeholder so I could get the URL for the deprecation code 😁